README.md

```diff
@@ -694,6 +694,17 @@ Your project is configured to generate types for the GraphQL APIs in use. Run `n
 the site locally to generate query-specific types. This is a big advantage of the TakeShape GraphQL mesh — you will have
 your queries and responses fully typed, and can develop efficiently and safely.
 
+## Playwright End-to-End (E2E) Testing
+
+### Local test development
+
+You can run and develop your Playwright tests locally using the following process:
+
+1. Edit your `.env.local` file and ensure your have the correct environment variables set for your personal project.
+1. Start a Next.js server, `npm run dev` will do.
+1. In another process (terminal) run `npm run playwright:ui`
+1. Work within the Playwright UI to run and debug your tests.
+
```

Comment:
Add a section here: `### CI` and list out all the `PLAYWRIGHT_` vars that can be set in the repo variables.